docs/quickstart-examples.md

```diff
@@ -0,0 +1,105 @@
+# Quickstart Examples
+
+Now that you have installed the basic requirements, you can start pulling data. Try out the examples below, or navigate to the [api/client/samples/](https://github.com/gro-intelligence/api-client/blob/development/api/client/samples) folder for more options.
+
+## Get Started in 120 Seconds
+
+If you have a google account, or if you already have jupyter notebooks and python installed, and a Gro API token, you can start pulling Gro data via our API client within 120 seconds.
+
+__Step 1: Get your API token__
+1. Login to your Gro web app account at [app.gro-intelligence.com](app.gro-intelligence.com)
+2. Open the Account button in the bottom left corner of the app.
+
+![account-button](./_images/account-button.PNG)
+
+3. Navigate to the API tab.
+
+![account-api-section](./_images/account-api-section.PNG)
+
+4. Copy your token.
+
+__Step 2: Run the code in a notebook__
+1. Navigate to [Google Colab](https://colab.research.google.com/), or launch jupyter notebooks from your command line with the command `jupyter notebook`. 

```

Comment:
"jupyter notebooks" -> "Jupyter" capitalized and singular

Reply:
Done